float64 x
float64 y
float64 t     # timestamp in seconds
float64 v     # speed at that point
